src/enzyme_ad/jax/BUILD

```diff
@@ -245,27 +251,92 @@ cc_library(
         ":mhlo-derivatives",
         ":stablehlo-derivatives",
         ":chlo-derivatives",
+        ":tensat_lib",
         "@enzyme//:EnzymeMLIR",
         "@llvm-project//llvm:Support",
+        "@llvm-project//mlir:AffineDialect",
+        "@llvm-project//mlir:AllPassesAndDialects",
```

Comment:
Shouldn't this already add all the passes and dialects? If so, why so many new deps then?